README.md

```diff
@@ -82,31 +82,31 @@ Hokusai currently supports Python 3.7+ only. The last version that supported Pyt
 
 ## Setup
 
-We assume that you already have Kubernetes cluster, Git, Docker, and Docker-Compose set up, and that you have an AWS account. Perform the following steps to setup Hokusai:
+We assume that your org admin has already set up a Kubernetes cluster and an AWS infrastructure, and that your local environment has Git, Docker, and Docker-Compose installed. Perform the following steps to get Hokusai working:
 
-1. [Configure AWS credentials](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#configuring-credentials).
+1. Ensure your org admin has completed the steps mentioned in [Administering Hokusai](./docs/Administering_Hokusai.md).
 
-2. Configure Hokusai
+2. Configure your local environment with [AWS credentials](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#configuring-credentials).
+
+3. Run Hokusai configure command:
 
     ```
-    hokusai configure --kubectl-version <kubectl version> --s3-bucket <bucket name> --s3-key <file key>
+    HOKUSAI_GLOBAL_CONFIG=<path-to-org-wide-global-config-file> hokusai configure
```

Comment:
It's a little unclear if `configure` can be run without a global config, or what defaults are imposed if so.

~Maybe an example config file could help, or an example of also passing args.~ Nevermind I see that's discussed below. (Still not sure if a global config is _required_ though.)

Reply:
[how now?](https://github.com/artsy/hokusai/pull/379/commits/8de9f317beca06b94608b8d2bb45f6d8fef459f3)

Reply:
👍